mne/utils/docs.py

```diff
@@ -1003,6 +1006,13 @@
         Support for reducing rank in all modes (previously only supported
         ``pick='max_power'`` with weight normalization).
 """
+docdict['check_passed'] = """
+check_passed : bool
+    If True, sanity-check passed MEG values against computed ones (only used
+    for covariances; default False).
```

Comment:
this is not explicit enough.

Reply:
can we find a better name than check_passed ? will it error or warn?

Reply:
It will warn. I guess we could rename it `on_mismatch : str` with raise/warn(default)/ignore like we do with `on_missing`.